<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionParameters</class>
 <widget class="QWidget" name="ConnectionParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConnectionParameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="controllTypeTab">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Internal</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>ModBus Parameters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
              <property name="sizeConstraint">
               <enum>QLayout::SetMaximumSize</enum>
              </property>
              <item>
               <widget class="QComboBox" name="comboBoxCOM"/>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Serial port</string>
                </property>
                <property name="buddy">
                 <cstring>comboBoxCOM</cstring>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
              <item>
               <widget class="QComboBox" name="comboBoxSpeed"/>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Transmition Speed</string>
                </property>
                <property name="buddy">
                 <cstring>comboBoxSpeed</cstring>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
              <item>
               <widget class="QSpinBox" name="spinBoxID">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>127</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Device ID</string>
                </property>
                <property name="buddy">
                 <cstring>spinBoxID</cstring>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="communicationErrorLabel">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>External</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>External IP adress :</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="IpLineEdit" name="ipEd0"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>.</string>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="IpLineEdit" name="ipEd1"/>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>.</string>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="IpLineEdit" name="ipEd2"/>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>.</string>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="IpLineEdit" name="ipEd3"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="externalStatusLabel">
           <property name="text">
            <string>Status</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="connectButton">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="disconnectButton">
             <property name="text">
              <string>Disconnect</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>IpLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">iplineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
